src/soorgeon/cli.py

```diff
@@ -74,3 +74,21 @@ def refactor(path, log, product_prefix, df_format, single_task, file_format):
 * Jupyter integration: https://ploomber.io/s/jupyter
 * Other editors: https://ploomber.io/s/editors
 """)
+
+
+@cli.command()
+@click.argument("filename", type=click.Path(exists=True))
+@click.option("--deep", "-d", default=None)
+def clean(filename, deep):
+    """
+    Clean a refactored notebook task.
+
+    $ soorgeon clean path/to/script.py
+    or
+    $ soorgeon clean path/to/notebook.ipynb
+
+    """
+    basic_clean(filename)
+    if deep:
```

Comment:
let's remove this since we haven't implemented it

Reply:
Will do. It would be better if it is pointed that out early in my implementation plan (under the issue)!